x/staking/legacy/v040/migrate.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
Should this file be edited? (Is it required to make tests pass?)

Reply:
++

This would probably need to go into a new v041 migrate